### PR TITLE
Fixes daturatea fruit

### PR DESCRIPTION
**Pull request by printer16**
## About The Pull Request

Fixes daturatea fruit so it correctly removes the spiritual trait after it leaves the body.

## Why It's Good For The Game

Prevents players from getting a free trait.

## Pre-Merge Checklist
- [X] Your Pull Request contains no breaking changes
- [ ] You tested your changes locally, and they work.
- [X] There are no new Runtimes that appear.
- [X] You documented all of your changes.

## Changelog
:cl:
fix: Fixes daturatea fruit from letting you get a free spiritual trait.
/:cl:

### CODE REVIEW

**Review by hecksadecimal — CHANGES_REQUESTED** [review]
Ensure you test changes locally, I can't merge this until the pre-merge checklist is complete.

**Comment by printer16** [comment]
the fruit itself doesn't work upon further investigation. will close and reopen later with a fix for that as well.

**Pull request closed without merging**
